pacehrh/R/pace_task_time.R

```diff
@@ -25,9 +25,15 @@ TaskTimes <- function(){
     n <- as.vector(prm$Total[, year][tasks$popRangeMaskPtr])
     p <- as.vector(pm[, year])
 
+    coverageRates <- filter(BVE$taskCoverageRates, years==year)
+    taskvals <- merge(x=taskvals, y=coverageRates, by.x="row.names", by.y="Indicator", all.x=TRUE)
+    
     # Compute the number of executed tasks ("service count")
     tasksN <- n * p * taskvals[,"RateMultiplier"] * (taskvals[,"NumContactsPerUnit"] + taskvals[,"NumContactsAnnual"])
-
+    
+    # Scale by the coverage rate
+    tasksN <- tasksN * taskvals[,"coverage"]
```

Comment:
I have some concerns with this approach.

I totally agree that this is the easiest place in the code to insert this calculation.

However ...

The challenge I'm seeing goes like this. There are two sides to the resource management story: the demand side (how much time it's going to take to supply the various healthcare services), and the supply side (how many resources are available to address the demand). I think it's important to the design of the system that we keep the demand and supply sides separate. Doing the coverage calculation this way conflates the two sides of the equation. It works in an algebra sense, but it's dubious as code architecture.

Let's discuss.

Reply:
This makes sense! As I traced through the code this seemed like the right place for it, but based on your explanation I agree, it's actually kind of a trap. Let me try again and see if it fits better.

Reply:
![image](https://github.com/InstituteforDiseaseModeling/PACE-HRH/assets/53099836/08bcb54c-d0cb-4fb1-a757-baad2a65d2a3)